Source/WTF/wtf/UUID.h

```diff
@@ -104,6 +109,7 @@ WTF_MAKE_FAST_ALLOCATED;
     bool isValid() const { return m_data != emptyValue && m_data != deletedValue; }
 
     UInt128 data() const { return m_data; }
+    void copyToUUID_T(uuid_t uuid) { memcpy(uuid, &m_data, 16); }
```

Comment:
I'd prefer to see sizeof(*uuid_t) instead of a magic number 16 here. For safety.

But also: what's the point of the memcpy? If we're sure the byte order and layout of UInt128 and uuid_t are the same, why can't we offer an operator uuid_t that just returns reinterpret_cast<uuid_t>(m_data)?

Reply:
> I'd prefer to see sizeof(uuid_t) instead of a magic number 16 here. For safety.
> 
> But also: what's the point of the memcpy? If we're sure the byte order and layout of UInt128 and uuid_t are the same, why can't we offer an operator uuid_t that just returns reinterpret_cast<uuid_t>(m_data)?

uuid_t is basically a typedef of char[16], so you can't make a return valid out of it.

But I'll improve this.